terminal_check_windows.go

```diff
@@ -1,4 +1,4 @@
-// +build !appengine,!js,windows
+// +build windows
```

Comment:
Now this change is truly not necessary, but I believe that's a nice-to-have drive-by cleanup. Feel free to revert it (by accepting this suggestion) if you don't like it.

```suggestion
// +build !appengine,!js,windows
```

Reply:
We need to keep the js tag, it is used to identify gopherjs build